netmanager-app/core/redux/slices/citiesSlice.ts

```diff
@@ -0,0 +1,97 @@
+// types.ts
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from "@reduxjs/toolkit";
+
+export interface SiteCategory {
+    tags: string[];
+    area_name: string;
+    category: string;
+    highway: string;
+    landuse: string;
+    latitude: number;
+    longitude: number;
+    natural: string;
+    search_radius: number;
+    waterway: string;
+  }
+  
+  export interface Site {
+    _id: string;
+    visibility: boolean;
+    images: string[];
+    land_use: string[];
+    isOnline: boolean;
+    formatted_name: string;
+    location_name: string;
+    search_name: string;
+    city: string;
+    district: string;
+    county: string;
+    region: string;
+    country: string;
+    name: string;
+    approximate_latitude: number;
+    approximate_longitude: number;
+    data_provider: string;
+    group: string;
+    site_category: SiteCategory;
+    groups: string[];
+    lastActive: string;
+  }
```

Comment:
If the type is going to be globally used then move it to the types folder @danielmarv